### ISSUE TITLE
TypeError: Object of type ChatStore is not JSON serializable

### ISSUE THREAD

**Issue by velecto1** [KnugiHK/Whatsapp-Chat-Exporter#37]
Hello, I got this error when running the current 'message_table' branch with '--json' option:

```
...
Gathering media...(4301/4301)
Gathering vCards...(6/6)
Copying media directory...
Traceback (most recent call last):
  File "/usr/local/bin/wtsexporter", line 8, in <module>
    sys.exit(main())
  File "/home/tomtom/.local/lib/python3.8/site-packages/Whatsapp_Chat_Exporter/__main__.py", line 222, in main
    data = json.dumps(data)
  File "/usr/lib/python3.8/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type ChatStore is not JSON serializable
```

**Comment by KnugiHK** [comment]
Fixed by #26 and commit 430a5ec. I will leave this open until it is released with #9.

**Comment by KnugiHK** [comment]
Released in [0.9.0](https://github.com/KnugiHK/Whatsapp-Chat-Exporter/releases/tag/0.9.0).
